octoprint_mrbeam/iobeam/laserhead_handler.py

```diff
@@ -251,7 +252,7 @@ def _calculate_power_correction_factor(self):
 
             # laserhead model S fix for correction factor
             # TODO fix this GOAL_MAX problem for all laser heads in a separate issue SW-394
-            if self._current_used_lh_model == "S":
+            if self._current_used_lh_model_string == "S":
```

Comment:
Also Please use constants instead of magic values.

Reply:
This is only there till SW-394 will fix this bug, so the constant won't be needed

Reply:
Why would sw-394 (ticket for the power correction settings) fix this? Why not fix it now?

Reply:
The power calibration is still not activated for the dreamcut devices, only for the [S] laser heads.
To fix it for the dreamcut laserheads we have to adjust the custom material settings or inform the customers that they have to check the settings.